miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/EditLibraryController.java

```diff
@@ -481,19 +481,8 @@ public JSONObject referenceDataJsonString() throws IOException {
   /**
    * Each PlatformName holds a null TagBarcodeFamily.
    *
-   * Structure of this tagBarcodes object: {
-   *   PlatformName : {
-   *     TagBarcodeFamilyName: {
-   *       1: [
-   *         {
-   *           id: ##,
-   *           name: AAAA,
-   *           sequence: XXXXX
-   *         }, ...
-   *       ], ...
-   *     }, ...
-   *   }, ...
-   * }
+   * Structure of this tagBarcodes object: { PlatformName : { TagBarcodeFamilyName: { 1: [ { id: ##, name: AAAA, sequence: XXXXX }, ... ],
+   * ... }, ... }, ... }
```

Comment:
Autoformat breaks intended formatting of Javadoc (though the Javadoc needs \<pre\> tags to use that formatting...)